perfkitbenchmarker/providers/aws/aws_emr.py

```diff
@@ -140,16 +151,18 @@ def _DeleteSecurityGroups(self):
     for group in manager_sg, worker_sg:
       cmd = self.cmd_prefix + ['ec2', 'delete-security-group',
                                '--group-id=' + group]
-      vm_util.IssueCommand(cmd)
+      vm_util.IssueRetryableCommand(cmd)
```

Comment:
Retryable commands should be idempotent - i suspect that this will fail if you try it  after the group has been deleted.